src/test/resources/features/3.7.x.feature

```diff
@@ -7,4 +7,13 @@ Feature: 3.7.x
     Examples: 
       | issueNumber | subtest | datasrc | args | expectation |
 
+#| 992 | 1 | "github992" | "-t {datasrc}/ff_del.xml" | "summary:totalErrors=1,summary:totalWarnings=5,summary:productValidation:failed=1,summary:messageTypes:error.table.field_value_data_type_mismatch=1,summary:messageTypes:warning.table.field_value_format_precision_mismatch=1,summary:messageTypes:warning.table.field_value_format_specifier_mismatch=4,summary:messageTypes:warning.table.field_value_too_long=1" |
```

Comment:
There is no error error and certainly should not be commented out. Ticket #1179 will either be resolved with an update to the pattern in the PDS4 documentation or determined to be correct as stands. Either way, its original form should remain until #1179 is resolved one way or the other.

Reply:
@al-niessner I think this test was copied from the other feature file just to get it started.